gratipay/billing/payday.py

```diff
@@ -27,7 +27,7 @@
 
 
 with open('sql/payday.sql') as f:
-    PAYDAY = f.read()
+    PAYDAY = f.read() % {'minimum_charge': MINIMUM_CHARGE}
```

Comment:
Cool. A further improvement could be to call it `payday_settings`, and create it as a temporary table as part of payday.